crates/cli/src/commands/run.rs

```diff
@@ -35,8 +35,8 @@ pub struct Command {
     dry_run: bool,
 
     /// Additional arguments to pass to the codemod
-    #[arg(last = true)]
-    params: Vec<String>,
+    #[arg(long)]
+    param: Option<Vec<String>>,
```

Comment:
```suggestion
    #[arg(long = "param", value_name = "KEY=VALUE")]
    params: Vec<String>,
```

Also why do we need to make it Option? doesn't it yield an empty vector by default?